libs/wingsdk/src/target-tf-aws/service.ts

```diff
@@ -0,0 +1,338 @@
+                  Action: "none:null",
+                  Resource: "*"
+                }
+              ]
+            })
+          }
+        }),
+      }]
+    });
+
+    let logGroup = new CloudwatchLogGroup(this, "LogGroup", {
+      name: `/ecs/${this.assetName}`,
+      retentionInDays: 7,
+    });
+
+    let taskDef = new EcsTaskDefinition(this, "Task", {
+      family: this.assetName,
+      executionRoleArn: executionRole.arn,
+      taskRoleArn: taskRole.arn,
+      containerDefinitions: Lazy.anyValue({
+        produce: () => JSON.stringify([
+            {
+              name: this.assetName,
+              image: image.name,
+              environment: Object.entries(this._env).map(([key, value]) => {
+                return {
+                  name: key,
+                  value: value,
+                };
+              }
+              ),
+              logConfiguration: {
+                logDriver: "awslogs",
+                options: {
+                  "awslogs-group": `/ecs/${this.assetName}`,
+                  "awslogs-region": "us-east-1", // TODO: Can I ignore this?
+                  "awslogs-stream-prefix": logGroup.name,
+                }
+              }
+          }
+        ])
+      }) as any,
+      networkMode: "awsvpc",
+      requiresCompatibilities: ["FARGATE"],
+      memory: "512",
+      cpu: "256",
+    });
+
+    let subnetIds = app.subnets["private"].map((subnet) => subnet.id);
+
+    let sg = new SecurityGroup(this, "SecurityGroup", {
+      vpcId: app.vpc.id,
+      egress: [
+        {
+          fromPort: 0,
+          toPort: 0,
+          protocol: "-1",
+          cidrBlocks: ["0.0.0.0/0"],
+        }
+      ],
+      ingress: [
+        { // TODO: Support network config
+          fromPort: 0,
+          toPort: 0,
+          protocol: "-1",
+          cidrBlocks: ["0.0.0.0/0"]
+        }
+      ]
+    });
+
+    this.service = new EcsService(this, "Service", {
+      name: this.assetName,
+      cluster: app.ecsCluster.arn,
+      desiredCount: 1,
+      launchType: "FARGATE",
+      taskDefinition: taskDef.arn,
+      forceNewDeployment: true,
+      enableExecuteCommand: true,
+      networkConfiguration: {
+        subnets: subnetIds,
+        securityGroups: [
+          sg.id
+        ]
+      }
+    });
+  }
+  
+  addPolicyStatements(...policies: PolicyStatement[]): void {
+    if (!this.policyStatments) {
+      this.policyStatments = [];
+    }
+
+    for (const policy of policies) {
+      this.policyStatments.push({
+        Action: policy.actions,
+        Effect: policy.effect ?? "Allow",
+        Resource: policy.resources
+      })
+    }
+  }
+
+  addNetwork(config: NetworkConfig): void {
+    // TODO: handle networking
+    console.log("Adding network", config);
+  }
+
+  _liftMap?: LiftMap | undefined;
+
+  /** @internal */
+  public _preSynthesize(): void {
+    super._preSynthesize();
+    let wrapper = `
+const service = require("${resolve(this.entrypoint)}");
```

Comment:
Eventually this should be extracted into some common helper so it can be used across different container providers, but that's fine for now.